src/App.tsx

```diff
@@ -71,7 +112,63 @@ const App = observer(() => {
   if (!store.isStarting) {
     return (
       <Layout>
-        <Header />
+        <Header handleOpenChat={handleOpenChat} />
+        {openChat.shouldOpenMaximizedChat === true && (
```

Comment:
`openChat.shouldOpenMaximizedChat` to **boolean**, nie trzeba tutaj porównywać z `true`, wystarczy samo: 
```suggestion
        {openChat.shouldOpenMaximizedChat && (
```